hooks/post-command

```diff
@@ -48,7 +56,7 @@ main() {
   ci_env=$(bash <(curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/env))
```

Comment:
also, we use this script, which is `codecov.io/env`, not `codecov.io/bash`...

Reply:
They don't provide a checksum for this.... Might be easier to inline though as its smaller

Reply:
actually I was wrong here, we use both scripts `/env` is executed on the buildkite agent, while `/bash` is executed inside a docker container.

Reply:
let's inline `/env` script and do checksum verification for `/bash` as you suggested